pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -62,15 +66,44 @@ func (f *Fit) Name() string {
 	return FitName
 }
 
+func validateFitArgs(args *schedulerv1alpha2.NodeResourcesFitArgs) error {
+	var allErrs field.ErrorList
+	resPath := field.NewPath("ignoredResources")
+	for i, res := range args.IgnoredResources {
+		path := resPath.Index(i)
+		if errs := metav1validation.ValidateLabelName(res, path); len(errs) != 0 {
+			allErrs = append(allErrs, errs...)
+		}
+	}
+
+	groupPath := field.NewPath("ignoredResourceGroups")
+	for i, group := range args.IgnoredResourceGroups {
+		path := groupPath.Index(i)
+		if errs := metav1validation.ValidateLabelName(group, path); len(errs) != 0 {
```

Comment:
Are both `example.com` and `example.com/` valid?

Reply:
`example.com` is valid and `example.com/` is not valid.

Reply:
Please add a test for this

Reply:
would this allow specifying `example.com/foo` in `IgnoredResourceGroups`? that wouldn't be effective, right?

Reply:
same question about whether `ValidateLabelName` is actually what we want here